commands.php

```diff
@@ -0,0 +1,31 @@
+<?php
+include "header.php";
+
+if(!isset($_SERVER['CONTENT_TYPE']) ||
+   $_SERVER['CONTENT_TYPE'] !== 'application/x-www-form-urlencoded') {
+    $log->warning("actions must be application/x-www-form-urlencoded, exiting.");
+    exit();
+}
+
+$params = [];
+foreach (explode('&', $request_body) as $chunk) {
```

Comment:
same suggestion as on the previous file (using `$_REQUEST` instead).
Also, if we'd rather keep this loop, since it's duplicated with the one on the other file, it could probably make more sense to extract it in a function

Reply:
Why `$_REQUEST` instead of `$_POST`?
The doc says:
- for commands: "A payload is sent via an HTTP POST request to your app."
- for actions: "[...], Slack will send a POST request to the request URL provided in the app settings."